PublishedModules/glueckself/mpdclient/mpdclient.install

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "mpdclient",
+	"version": 0.1,
+	"author": "glueckself",
+	"maintainers": ["glueckself"],
+	"desc": "Control an MPD server",
+	"aliceMinVersion": 0.10,
+	"systemRequirements": [],
+	"pipRequirements": ["python-mpd2"],
+	"conditions": {
+		"lang": [
+			"en",
+			"de"
+		],
+		"online": true
```

Comment:
This means the user HAS to allow internet usage, MPD is local no? In which case you don't need that condition at all, we don't care if the user is online, offline, allows or forbids connectivity etc etc

Reply:
I get it now. Thank you!
MPD can local or remote. I guess it will be usually in the same network as Alice (my case) or maybe a hop or two away.